home-application/src/main/java/com/softserveinc/ita/homeproject/application/service/home/HomeUserDetailsService.java

```diff
@@ -1,13 +1,13 @@
-package com.softserveinc.ita.homeproject.application.service.impl;
```

Comment:
I think it would be more correct to create a package "security" in the package "config", into which to transfer the HomeUserDetailsService, HomeUserWrapperDetails and SecurityConfig classes. And the package "home" is not needed.